Classes

```diff
@@ -1,6 +1,6 @@
 1)Class-Player // main player(-s) class
 2)Class-Bot  // bot class
-3)Class-Unit // in-game units class
```

Comment:
I think it would be logical to name class Units because, there are not 1 unit.

Reply:
Let's discuss it. One object of this class will be one unit or group of units? If class describes a single unit, then it should be Unit.

Reply:
It can be some general class for game play, like unit selection.it can be 1 or a lot.